tests/test_axis_limits.py

```diff
@@ -395,7 +364,8 @@ def get_source(name):
     return unsafe_names
 
 
-@pytest.mark.unit
+@pytest.mark.slow
+@pytest.mark.regression
```

Comment:
It's kind of demoralizing run `pytest -m unit`  and wait on `tests_axis_limits`  so I changed this to regression